python/social_bot/models/pioneer2dx_noplugin/model.sdf

```diff
@@ -188,7 +188,6 @@
         <xyz>0 0 1</xyz>
         <use_parent_model_frame>true</use_parent_model_frame>
       </axis>
-      <dynamics damping="0.01" friction="0.1"/>
```

Comment:
This throws a warning, as fixed joint does not work with dynamics option. So removed.